airflow/cli/cli_parser.py

```diff
@@ -626,6 +626,20 @@ def positive_int(value):
     ),
     action="store_true",
     default=False)
+ARG_CONN_IMPORT = Arg(
+    ('file',),
+    help='Import connections from a file. Acceptable file formats .json, .yaml, .env',
+    type=str)
+ARG_CONFLICT_DISPOSITION = Arg(
+    ('--conflict-disposition',),
+    help=("Specifies the action that occurs if the connection already exists. Default value : restrict\n"
+          "Values -\n"
+          "\toverwrite : Overwrites the connection\n"
+          "\tignore : Skip this connection\n"
+          "\trestrict : Raise an exception\n"),
+    type=str)
```

Comment:
Can you consider to use ``choices`` parameter?
https://docs.python.org/3/library/argparse.html#choices

Reply:
Updated code.